test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetRestoreCommandTest.cs

```diff
@@ -2747,82 +2737,6 @@ public void RestoreCommand_PackageNamespaceLongerMatches_Succeed()
             }
         }
 
-
-        [Fact]
-        public void RestoreCommand_PackageNamespace_NoNamespaceMatches()
-        {
-            // Arrange
-            var nugetexe = Util.GetNuGetExePath();
-
-            using (var workingPath = TestDirectory.Create())
-            {
-                var proj1Directory = Path.Combine(workingPath, "proj1");
-                Directory.CreateDirectory(proj1Directory);
-
-                var proj1File = Path.Combine(proj1Directory, "proj1.csproj");
-                File.WriteAllText(
-                    proj1File,
-                    @"<Project ToolsVersion='4.0' DefaultTargets='Build'
-    xmlns='http://schemas.microsoft.com/developer/msbuild/2003'>
-  <PropertyGroup>
-    <OutputType>Library</OutputType>
-    <OutputPath>out</OutputPath>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
-  </PropertyGroup>
-  <ItemGroup>
-    <None Include='packages.config' />
-  </ItemGroup>
-</Project>");
-
-                var sharedRepositoryPath = Path.Combine(workingPath, "SharedRepository");
-                Directory.CreateDirectory(sharedRepositoryPath);
-                Util.CreateTestPackage("My.MVC.ASP", "1.0.0", sharedRepositoryPath);
-
-                Util.CreateFile(proj1Directory, "packages.config",
-@"<packages>
-  <package id=""My.MVC.ASP"" version=""1.0.0"" targetFramework=""net461"" />
-</packages>");
-
-                var configPath = Path.Combine(workingPath, "nuget.config");
-                SettingsTestUtils.CreateConfigurationFile(configPath, $@"<?xml version=""1.0"" encoding=""utf-8""?>
-<configuration>
-    <packageSources>
-    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
-    <clear />
-    <add key=""SharedRepository"" value=""{sharedRepositoryPath}"" />
-    </packageSources>
-    <packageNamespaces>
-        <packageSource key=""SharedRepository"">
-            <namespace id=""Contoso.MVC.ASP"" />  <!-- My.MVC.ASP doesn't match-->
-        </packageSource>
-    </packageNamespaces>
-</configuration>");
-
-                var packagePath = Path.Combine(workingPath, "packages");
-
-                string[] args = new string[]
-                    {
-                        "restore",
-                        proj1File,
-                        "-solutionDir",
-                        workingPath,
-                        "-Verbosity",
-                        "d"
-                    };
-
-                // Act
-                var r = CommandRunner.Run(
-                    nugetexe,
-                    workingPath,
-                    string.Join(" ", args),
-                    waitForExit: true);
-
-                // Assert
-                Assert.Equal(_successCode, r.ExitCode);
```

Comment:
Should we verify a failure here instead of deleting the test?

Reply:
I agree with Nikolche. Unless I am missing something I don't see a test in this PR that asserts for the restore failure.

Reply:
@nkolev92 
Ok. Added back the tests with verification of a failure.

Reply:
When the failure occurs, it is obvious why it's happening?

Is there an actual error logged?

Reply:
May be assert `result.Errors.Count` to ensure errors are logged.

Reply:
> When the failure occurs, it is obvious why it's happening?
> 
> Is there an actual error logged?

@nkolev92 
Added error details. Please check now.